Source/Wit/Public/Wit/Utilities/WitHelperUtilities.cpp

```diff
@@ -321,54 +322,92 @@ FString FWitHelperUtilities::GetVoiceClipId(const FTtsConfiguration& ClipSetting
  * @param RawDataSize [in] the raw data size
  * @return the sound wave created or null if unsuccessful
  */
-USoundWave* FWitHelperUtilities::CreateSoundWaveFromRawData(const uint8* RawData, const int32 RawDataSize)
+USoundWave* FWitHelperUtilities::CreateSoundWaveFromRawData(
+	const uint8* RawData,
+	const int32 RawDataSize,
+	const EWitRequestAudioFormat AudioFormat,
+	const bool bStream)
 {
-	FWaveModInfo WaveInfo;
-	const bool bIsValidWaveInfo = WaveInfo.ReadWaveInfo(RawData, RawDataSize);	
-
-	if (!bIsValidWaveInfo)
+	FSoundWaveParams SoundWaveParams = FSoundWaveParams();
+	USoundWave* SoundWave;
+	switch (AudioFormat)
+	{
+	case EWitRequestAudioFormat::Wav:
 	{
```

Comment:
Minor: indentation. Keep it consistent with Source/Wit/Private/Wit/Request/WitRequestBuilder.cpp